conf/scylla.yaml

```diff
@@ -534,3 +534,8 @@ murmur3_partitioner_ignore_msb_bits: 12
 # queries correctly.
 # The option can be updated without restarting Scylla.
 # reversed_reads_auto_bypass_cache: true
+
+# Do not use optimized query path (which opens sstables lazily) when querying
+# TWCS-compacted sstables. Use the same method that is used for other compaction
+# strategies.
+disable_optimized_twcs_queries: true
```

Comment:
We prefer not to use like negative terms in flags, because it leads to double negation (`disable: false`) which is confusing. Better call this `enable_optimized_twcs_queries` and default it to `true`.

Reply:
Done